### DISCUSSION TITLE
TableFooterCell 支持显示节点阈值 ShownWithBreakPoint

### DISCUSSION THREAD

**Discussion by densen2014** [dotnetcore/BootstrapBlazor#3693] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

因为表体 TableColumn 写了  
```
<TableColumn @bind-Field="@context.Cash" Text="@Localizer["Cash"]" FormatString="N2" Align="Alignment.Center" ShownWithBreakPoint="BreakPoint.Large" />
```
所以希望合计栏目也统一风格, 响应式隐藏相关字段

**Comment by bb-auto** [comment]
@densen2014 Thank you for contacting us. We will give feedback later.

**Reply by ArgoZhang** [reply]
@densen2014 这个问题设计的时候也考虑，不太好弄，`Footer` 基本上使用模板或者设置 `colspan` 属性来设置，所以无法适应，需要重新设计其逻辑

**Comment by ArgoZhang** [comment]
@densen2014 

### `TableFooterCell` 增加 `ColspanCallback` 回调方法

```
[CascadingParameter(Name = "TableBreakPoint")]
private BreakPoint BreakPoint { get; set; }
```

页面变化时，及时通知 `TableFooterCell` 控制不同宽度需要合并数量即可解决，示例代码如下

Razor
```
<TableFooterCell Text="@Localizer["TablesFooterStatisticsTotal"]" ColspanCallback="ColspanCallback" Align="@Align" />
```

C#
```
private static int ColspanCallback(BreakPoint breakPoint) => breakPoint == BreakPoint.None ? 0 : 3;
```
